config/genesis/genesis.go

```diff
@@ -15,6 +17,15 @@ type Genesis struct {
 	Genesis    GenesisFields
 }
 
+// Genesis stores the data parsed from the genesis configuration file
+type GenesisData struct {
+	Name          string
+	Id            string
+	Bootnodes     [][]byte
```

Comment:
Why is this `[]byte`?

Reply:
`[]string` inside a struct still doesn't work with scale :/

Reply:
Please open an issue

Reply:
done https://github.com/ChainSafe/gossamer/issues/394